plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/internal/RefreshVersionsConfigHolder.kt

```diff
@@ -55,6 +57,11 @@ object RefreshVersionsConfigHolder {
         )
         .build()
 
+    internal val moshi : Moshi =
```

Comment:
I'll merge `develop` back here and use a resettable delegate (introduced in #244) to minimize the risks of leaking memory in the Gradle daemon.

Reply:
I figured we don't need to keep a moshi instance around at all.

The `RefreshVersionsConfigHolder` is instantiated once per build (Gradle makes a new classloader on every new build), but that `moshi` instance is needed only if we check Gradle updates (i.e. when the `refreshVersions` task is run).